addons/sale/controllers/combo_configurator.py

```diff
@@ -50,11 +50,17 @@ def sale_combo_configurator_get_data(
         pricelist = request.env['product.pricelist'].browse(pricelist_id)
         date = datetime.fromisoformat(date)
         selected_combo_item_dict = {item['id']: item for item in selected_combo_items or []}
+        website_request = request.get('website')
```

Comment:
`request` is not a dict and there has no attribute `get()`. 

You can use the following suggestion or change the override in `website_sale`.
```suggestion
        website_request = request.is_frontend and request.website
```